docs/developer-reference/contracts/error-signatures.md

```diff
@@ -6,132 +6,261 @@ order: 4
+| `DerivedTauXWrong()`                              | `0x4071c5a8` |                                                                                         | MockGyroECLPPool                |
+| `DerivedUWrong()`                                 | `0xf84d4b44` |                                                                                         | MockGyroECLPPool                |
+| `DerivedVWrong()`                                 | `0xcfb498d5` |                                                                                         | MockGyroECLPPool                |
+| `DerivedWWrong()`                                 | `0x83446b36` |                                                                                         | MockGyroECLPPool                |
+| `DerivedZWrong()`                                 | `0x12e3e411` |                                                                                         | MockGyroECLPPool                |
+| `InvariantDenominatorWrong()`                     | `0xd1c17993` |                                                                                         | MockGyroECLPPool                |
+| `MaxAssetsExceeded()`                             | `0x2da2a5e5` |                                                                                         | MockGyroECLPPool                |
+| `MaxInvariantExceeded()`                          | `0xdc10196f` |                                                                                         | MockGyroECLPPool                |
+| `DerivedDsqWrong()`                               | `0xfb154af0` |                                                                                         | MockGyroECLPPool                |
+| `AddOverflow()`                                   | `0xa7f965e3` |                                                                                         | MockGyroECLPPool                |
+| `SubOverflow()`                                   | `0x8a5d6af4` |                                                                                         | MockGyroECLPPool                |
+| `MulOverflow()`                                   | `0x0cde6c26` |                                                                                         | MockGyroECLPPool                |
+| `DivInterval()`                                   | `0xe03f5d57` |                                                                                         | MockGyroECLPPool                |
+| `OwnableUnauthorizedAccount(address)`             | `0x118cdaa7` | address account                                                                         | MockLBPool                      |
+| `OwnableInvalidOwner(address)`                    | `0x1e4fbdf7` | address owner                                                                           | MockLBPool                      |
```

Comment:
"InvalidOwner" is in the LBPPoolFactory; not sure where it's getting the "Ownable" from. We don't use that OZ contract directly (with LBPs, anyway; some of the hooks do).